nut/config.yaml

```diff
@@ -51,6 +51,7 @@ schema:
     - name: str
       driver: str
       port: str
+      powervalue: list(0|1)?
```

Comment:
Why not make this a boolean?

Reply:
oh, actually..... it should just be an integer. It's possible for that value to be 2 or 3 on servers with many PSUs (I doubt anyone would use this for their HA server, but, you never know..) 
Will update